CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+## Release 5.0.0
+
+### Summary
+
+This major release includes a major internal refactoring and code optimizations around the splunk types and providers as well as changes to how resource purging is handled.  This release should be mostly backwardly compatible with the only notable API change being the title_patterns parsing of resource titles and namevar differences.
```

Comment:
please mention that it is the first release in our namespace and where it came from